### PR TITLE
Add guide for Action Mailer Previews

### PR DESCRIPTION
**Pull request by yuki24**
We really should add this to Action Mailer Basics.

cc/ @zzak

### CODE REVIEW

File: guides/source/action_mailer_basics.md [hunk 1]
```diff
@@ -442,6 +442,39 @@ end
...
+
+```ruby
+class UserMailerPreview < ActionMailer::Preview
+  def welcome_email
+    UserMailer.welcome_email(User.first)
+  end
+end
+```
+
+Then the preview will be available in http://localhost:3000/rails/mailers/user_mailer/welcome_email.
```
Comment (vipulnsward) [inline guides/source/action_mailer_basics.md]: Should be wrapped as <http://localhost:3000/rails/mailers/user_mailer/welcome_email>

File: guides/source/action_mailer_basics.md [hunk 2]
```diff
@@ -442,6 +442,39 @@ end
...
+
+If you change something in `app/views/user_mailer/welcome_email.html.erb`
+or the mailer itself, it'll automatically reload and render it so you can
+visually see the new style instantly. A list of previews are also available
+in http://localhost:3000/rails/mailers.
```
Comment (vipulnsward) [inline guides/source/action_mailer_basics.md]: same here

**Comment by yuki24** [comment]
updated commit. Thanks @vipulnsward!

**Comment by andyatkinson** [comment]
@yuki24 Looks good to me. We have two mailer previews in my work project in `test/mailers/previews/`. I looked at them following the instructions, double checking paths etc. :+1:

**Pull request merged**